README.md

```diff
@@ -565,7 +565,8 @@ GPG keys used to sign Node.js releases:
 * **Shelley Vohr** &lt;shelley.vohr@gmail.com&gt;
 `B9E2F5981AA6E0CD28160D9FF13993A75599653C`
 
-To import the full set of trusted release keys:
+To avoid nuances involved in verification of a sub-key possibly used to sign a
+release, import the full set of trusted release keys:
```

Comment:
```suggest
To import the full set of trusted release keys (including subkeys used to sign releases):
```

I think the current test has a bit too much editorialization. I think suggested text above accomplishes the intention while keeping things succinct.

LMK if you have alternative suggestions

Reply:
> ```
> To import the full set of trusted release keys (including subkeys used to sign releases):
> ```
IMHO, there is a need to mention something like "nuance"s or "pitfall"s, to make sure that users get an idea that if they choose to not import all the keys following (assuming #32654 is not going to land), they might (well) run into something they aren't expecting. My motivation was to prevent confusion & misuse of time (which happened to me).
So between this suggestion & the current version, i'd prefer the current version (because it's more informative, etc.).

P.S. It seems there is an agreement on a need for change, & hopefully there will be an agreement on the wording w/o spending too much time on it. For me, the current wording committed is better that the status quo.

Reply:
Ping @Trott @MylesBorins

Reply:
I stand by my original review. I think the current text is too editorialized and prefer the proposed text I suggested. I'm open to reviewing alternative suggestions